.github/workflows/cachix.yml

```diff
@@ -0,0 +1,28 @@
+# Publish the Nix flake outputs to Cachix
+name: Cachix
+on:
+  push: master
+
+jobs:
+  publish:
+    name: Publish Flake
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout sources
+      uses: actions/checkout@v2
+      with:
+        submodules: true
```

Comment:
Do you 100% need this? IIRC it looked like the flake hardcodes a snapshot of the grammars anyway.

Reply:
ah good call, probably don't need this :+1: